marconi-chain-index/test/Spec/Golden/CLIInputValidation/invalidAddress-MissingBech32Prefix.golden

```diff
@@ -2,8 +2,8 @@ option --addresses-to-index: Invalid address (not a valid Bech32 address represe
 
 Usage: marconi-chain-index [--version] (-s|--socket-path FILE-PATH) 
                            (--mainnet | --testnet-magic NATURAL) 
-                           [(-n|--slot-no SLOT-NO)
-                             (-b|--block-header-hash BLOCK-HEADER-HASH)] 
+                           [--start-from-genesis | 
+                             --start-from-last-sync-points | COMMAND] 
```

Comment:
What's that `COMMAND`? Can we put the actual expected data like `(--slotNo SLOTNO --block-header-hash BHH)`

Reply:
Yep, but it looks like I have to do it manually with `metavar`.

Reply:
the `| COMMAND` part is confusing, it sounds like I can use `--start-from-genesis`, `--start-from-last-sync-points` or ANY command.